python/sparkdl/graph/builder.py

```diff
@@ -75,6 +78,48 @@ def run(self, *args, **kargs):
         """
         return self.sess.run(*args, **kargs)
 
+    def asUDF(self, udf_name, fetches, feed_dict=None, blocked=False, register=True):
```

Comment:
Is this a comment suggesting action for this PR, or a comment to leave a comment here? 😅